src/wp-admin/includes/ajax-actions.php

```diff
@@ -2487,7 +2487,7 @@ function wp_ajax_delete_inactive_widgets() {
 		$id_base      = implode( '-', $pieces );
 		$widget       = get_option( 'widget_' . $id_base );
 		unset( $widget[ $multi_number ] );
-		update_option( 'widget_' . $id_base, $widget );
+		update_option( 'widget_' . $id_base, $widget, false );
```

Comment:
Let's leave this change for https://core.trac.wordpress.org/ticket/42743. It is its own problem to resolve and would make this PR more complicated.

Reply:
I agree